app/components/Issues/Detail/styledComponents.js

```diff
@@ -312,3 +312,8 @@ export const UsernameLink = styled(Link)`
     color: ${hoverLinkColor};
   }
 `;
+
+export const DeletedName = styled.div`
```

Comment:
If we go with the above approach, we can removed this and adjust the styling of `UsernameLink` to the following:
```
export const UsernameLink = styled(({ isUserDeleted, ...restProps }) => (
  <Link {...restProps} />
))`
  display: inline;
  font-weight: bold;
  pointer-events: ${({ isUserDeleted}) => isUserDeleted ? 'none' : 'inherit'};

  &:hover {
    text-decoration: underline;
  }
`;
```